docs/ref/models/fields.txt

```diff
@@ -628,8 +632,8 @@ uses :class:`~django.core.validators.EmailValidator` to validate the input.
 A file-upload field.
 
 .. note::
-    The ``primary_key`` and ``unique`` arguments are not supported, and will
-    raise a ``TypeError`` if used.
+    The ``primary_key`` argument isn't supported and will raise a
```

Comment:
Same comment about line length and actually TypeError is wrong I think (it's actually a system check error). "will raise an error if used." should be fine.